src/java/org/apache/cassandra/concurrent/InfiniteLoopExecutor.java

```diff
@@ -81,4 +83,10 @@ public boolean awaitTermination(long time, TimeUnit unit) throws InterruptedExce
         thread.join(unit.toMillis(time));
         return !thread.isAlive();
     }
+
+    @VisibleForTesting
+    public boolean isAlive()
+    {
+        return this.thread.isAlive();
```

Comment:
`this` is not needed here:
```suggestion
        return thread.isAlive();
```